crates/relayer-framework-test/src/relayer_mock/base/types/aliases.rs

```diff
@@ -0,0 +1,17 @@
+use std::collections::HashMap;
+
+use ibc_relayer_framework::base::one_for_all::traits::chain::OfaChainTypes;
+
+use crate::relayer_mock;
+use crate::relayer_mock::base::types::height::Height as MockHeight;
+
+pub type PacketUID = (PortId, ChannelId, Sequence);
+pub type ConsensusState =
+    <relayer_mock::contexts::chain::MockChainContext as OfaChainTypes>::ConsensusState;
+pub type ChainState = HashMap<MockHeight, ConsensusState>;
+pub type ClientId = <relayer_mock::contexts::chain::MockChainContext as OfaChainTypes>::ClientId;
```

Comment:
Here we should define the types directly such as:

```rust
type ConsensusState = State;
type ClientId = String;
```

And then inside the implementation of `OfaChainTypes`, we will refer to these type aliases.